src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
```

Comment:
in the case of the second example vs the first, wouldn't that impact the size of the final bundle if someone was to cherry pick a single component for a project ?
```jsx
import Button from 'stardust/elements/Button'
```